pkg/lib/bundle/generate.go

```diff
@@ -12,6 +12,7 @@ import (
 	"gopkg.in/yaml.v2"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	k8syaml "k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/kubectl/pkg/util/slice"
```

Comment:
Is this external dependency really necessary? In general, I think that utils like this which are not included in the standard library and are trivial to write should just be implemented directly.

Reply:
yeah, i see your point, just loved that library. getting rid of it.